db/migrate/20201008112050_add_content_to_article.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentToArticle < ActiveRecord::Migration[6.0]
+  def change
+    add_column :articles, :content, :text
+  end
+end
```

Comment:
Since you are not using the content for the index action then this is premature assumptions and out of scope